api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -127,22 +130,36 @@ public boolean isNotebookBlob(Blob blob) {
   @Override
   public FileDetail copyNotebook(
       String fromWorkspaceNamespace,
-      String fromWorkspaceName,
+      String fromWorkspaceFirecloudName,
```

Comment:
These parameters are the Workspaces' Firecloud/Terra names rather than their AoU names